# Mean, median and mode Background:
You have a sample of 11 people and their personal annual income. 

In [14]:
income = [62000.00, 64000.00,49000.00,324000.00,1264000.00,54330.00,64000.00,51000.00,55000.00,48000.00,53000.00]

## Task1: 
Calculate the mean, median and mode

In [15]:
import statistics as st

In [16]:
#Annual income mean
mean = st.mean(income)
print("The annual income mean is = ",round(mean,2))

The annual income mean is =  189848.18


In [17]:
#Annual income median
median = st.median(income)
print("The annual income median is = ",median)

The annual income median is =  55000.0


In [18]:
#Annual income mode
mode = st.mode(income)
print("The annual income mode is = ",mode)

The annual income mode is =  64000.0


## Task2:
Try to interpret on the numbers you got

### R:
- The mean is 189848.18$, this means that if all the money is share equally everyone will receive this amount.

- The median is 55000$ which means that the 50% get 55000$ or less in the annual income.

- The mode is 64000$ this means that the most common value for the annual income is 64000$.